src/FsAutoComplete/LspHelpers.fs

```diff
@@ -131,11 +131,13 @@ module Conversions =
         { Uri = uri
           Range = fcsRangeToLsp decl.Range }
 
-      let sym =
-        { SymbolInformation.Name = decl.LogicalName
+      let sym: SymbolInformation =
+        { Name = decl.LogicalName
           Kind = kind
           Location = location
-          ContainerName = container }
+          ContainerName = container
+          Tags = None
+          Deprecated = None }
```

Comment:
potential functionality gap here - we don't have information at the navigation symbol level to know if the symbol is deprecated or not.

Reply:
Where would we gather that info? Looks like we have full typechecks happening in AdaptiveServer before we get the declarations (looks like a performance oversight on my part since it only requires parse results).

Reply:
I'm not actually sure where that info surfaces in the API - I just know it's not in the `Navigation Item` APIs :D